### PR TITLE
Adding windows batch file to complement the bash script.

### PR DESCRIPTION
**Pull request by TheBosZ**

### CODE REVIEW

**Comment by TheBosZ** [comment]
It's hard to test batch files in windows. But this seems to get close to the expected behavior.

File: bin/flutter.bat [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+SET script_path=%flutter_tools_dir%\bin\sky_tools.dart
+REM TODO: Don't require dart to be on the user's path
+SET dart=dart
+
+REM Getting modified timestamps in a batch file is ... troublesome
+REM More info: http://stackoverflow.com/questions/1687014/how-do-i-compare-timestamps-of-files-in-a-dos-batch-script
+FOR %%f IN (%flutter_tools_dir%\pubspec.yaml) DO SET yamlt=%%~tf
+FOR %%a IN (%flutter_tools_dir%\pubspec.lock) DO SET lockt=%%~ta
+if !lockt! LSS !yamlt! (
+    CD "%flutter_tools_dir%"
```
Comment (abarth) [inline bin/flutter.bat]: Do we need to return to the current working directory or does that happen automatically?

**Comment by abarth** [comment]
LGTM

File: bin/flutter.bat (hunk 1 above)
Comment (TheBosZ) [inline bin/flutter.bat]: Good point. Just pushed an update to return to the flutter_root

File: bin/flutter.bat [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    CALL pub.bat get
+	CD "%flutter_root%"
```
Comment (abarth) [inline bin/flutter.bat]: Looks like you're mixing tabs and spaces.

**Comment by TheBosZ** [comment]
Stupid Notepad++ not knowing what I want!

**Comment by abarth** [comment]
Thanks!

**Comment by TheBosZ** [comment]
:+1:

**Pull request merged**